tensorboard/plugins/pr_curve/tf_pr_curve_dashboard/tf-pr-curve-steps-selector.html

```diff
@@ -87,14 +87,17 @@
       },
       timeDisplayType: String,
       // A mapping between run and slider index.
-      _runToStepIndex: Object,
+      _runToStepIndex: {
+        type: Object,
+        value: {},
```

Comment:
`value: () => ({}),` to not share an object across instances.

Reply:
Done.